scripts/cpplint.py

```diff
@@ -5024,6 +5024,21 @@ def CheckIncludeLine(filename, clean_lines, linenum, include_state, error):
     elif not _THIRD_PARTY_HEADERS_PATTERN.match(include):
       include_state.include_list[-1].append((include, linenum))
 
+    # Check module dependencies
+    module_name = os.path.dirname(filename)
+    has_src = module_name.find('src/')
+    if has_src >= 0:
+        module_name = module_name[has_src+4:]
```

Comment:
Should this be a warning as well?

Reply:
I don't think it's worth a warning. It will only affect the module name printed in the error message.